### PR TITLE
Review PBT impeller examples

### PR DESCRIPTION
**Pull request by oguevremont**
In preparation for the next release, examples are being checked and tested. This PR addresses the examples related to mixing using immersed boundaries (Nitsche or Sharp) to represent a PBT impeller.

### Description

All the examples were tested. They function properly and are consistent with one another.

### Checklist (will be removed when merged)
See [this page](https://chaos-polymtl.github.io/lethe/documentation/contributing.html#pull-requests) for more information about the pull request process.

Code related list:
- [x] Lethe documentation is up to date
- [x] Copyright headers are present and up to date
- [x] The branch is rebased onto master
- [x] Code is indented with indent-all and .prm files (examples and tests) with prm-indent
- [x] Links are added to parent .rst files
- [x] The example is following the [standard format](https://chaos-polymtl.github.io/lethe/documentation/contributing.html#general-rules-and-format)

Pull request related list:
- [x] Labels are applied
- [ ] There are at least 2 reviewers (or 1 if small feature) excluding the responsible for the merge
- [x] If this PR closes an issue or is related to a project, it is linked in the "Projects" or "Development" section
- [ ] If any future works is planned, an issue is opened
- [ ] The PR description is cleaned and ready for merge

### CODE REVIEW

File: doc/source/examples/sharp-immersed-boundary/3d-opencascade-mixer-with-pbt-impeller/3d-opencascade-mixer-with-pbt-impeller.rst [hunk 1]
```diff
@@ -11,34 +11,34 @@ Features
...
 ----------------------------
-Files Used in This Example
+Files Used in this Example
 ----------------------------
 
 Both files mentioned below are located in the example's folder (``examples/sharp-immersed-boundary/3d-opencascade-mixer-with-pbt-impeller``).
 
 - Parameter file: ``mixer.prm``
-- Step file: ``impeller.step``
+- `Step` file: ``impeller.step``
```
Comment (blaisb) [inline doc/source/examples/sharp-immersed-boundary/3d-opencascade-mixer-with-pbt-impeller/3d-opencascade-mixer-with-pbt-impeller.rst:24]:
```suggestion
- ``.step`` file: ``impeller.step``
```

**Review by blaisb — APPROVED** [review]
Some small changes, nothing big

File: doc/source/examples/sharp-immersed-boundary/inclined-3d-composite-mixer-with-pbt-impeller/inclined-3d-composite-mixer-with-pbt-impeller.rst [hunk 2]
```diff
@@ -267,7 +267,8 @@ The parameters used to define the impeller are based on the :doc:`../3d-composit
       end
     end
 
-The only noticeable differences are that the initial refinement and the refinement zone are adjusted respectively to 6 and 0 to 1.25 reference length. These values are chosen to guarantee that the refinement zone is big enough to cover the motion of the impeller and avoid interaction of the hanging nodes with the sharp immersed boundary constraints.
+Note that ``orientation`` and ``omega`` result from the previous calculations.
```
[the remark this answers is not in the document]
Comment (oguevremont) [reply doc/source/examples/sharp-immersed-boundary/inclined-3d-composite-mixer-with-pbt-impeller/inclined-3d-composite-mixer-with-pbt-impeller.rst]: I specified which calculations

**Comment by oguevremont** [comment]
I addressed the comments

**Pull request merged**